app/main.py

```diff
@@ -1 +1,5 @@
-# write your code here
+def copy_file(command: str) -> None:
+    for part in command.split(" "):
```

Comment:
The `command.split(" ")` should be checked to ensure it contains exactly three parts: the command 'cp', the source file name, and the destination file name. This is necessary to validate the input format as per the checklist.